illustrations/guidelines/index.md

```diff
@@ -57,26 +58,32 @@ Spot hero illustrations should be Rectangular artboard at **1024px x 768px.**
 **Azure** can be used at **100%** & **40%** opacity.
 **Prince** can be used at **40%** opacity only.
 
-<div class="u-clearfix m-top-6 m-bottom-6">
-	<div class="g-1_2 g-omega g-1_2__l bg-c-b300 p-top-12 m-bottom-2 m-bottom-0__l">
-		<ul class="t-center p-2 u-clearfix">
-			<li class="t-center t-c-w100 g-1_4"><span class="t-1">A</span><br />&#128515;</li>
-			<li class="t-center t-c-w100 g-1_4"><span class="t-0">A</span><br />&#128515;</li>
-			<li class="t-center g-1_4 t-c-b700"><span class="t-1">A</span><br />&#128515;</li>
-			<li class="t-center g-1_4 t-c-b700"><span class="t-0">A</span><br />&#128515;</li>
-		</ul>
-		<p class="p-3 bg-c-g100 m-0"><strong>Azure</strong><br />Sass $c-b300<br />Hex #0073EC<br />rgb(0,175,239)</p>
-	</div><!--Azure-->
-	<div class="g-1_2 g-1_2__l bg-c-p300 p-top-12 m-bottom-2 m-bottom-0__l">
-		<ul class="t-center p-2 u-clearfix">
-			<li class="t-center t-c-w100 g-1_4"><span class="t-1">A</span><br />&#128515;</li>
-			<li class="t-center t-c-w100 g-1_4"><span class="t-0">A</span><br />&#128515;</li>
-			<li class="t-center g-1_4 t-c-b700"><span class="t-1">A</span><br />&#128515;</li>
-			<li class="t-center g-1_4 t-c-b700"><span class="t-0">A</span><br />&#128515;</li>
-		</ul>
-		<p class="p-3 bg-c-g100 m-0"><strong>Prince</strong><br />Sass $c-p300<br />Hex #A644E5<br />rgb(166,68,229)</p>
-	</div><!--Prince-->
-</div><!--clearfix-->
+<div class="conatiner m-top-6 m-bottom-6">
```

Comment:
container class is spelled wrong (and might need to be `container-fluid` instead). Also you can use the shortcut `m-y-6` to add top and bottom margin instead of having to set both manually.